assets/js/script.js

```diff
@@ -189,3 +189,13 @@ window.addEventListener('load', () => {
 
 // Form validation
 
+form.addEventListener('submit', (event) => {
+  if (email.value.toLowerCase() !== email.value) {
+    event.preventDefault();
+    const emailError = document.querySelector('#email-error');
+    emailError.textContent = 'The email must be in lowercase letters';
+    emailError.classList.remove('hide');
+  } else {
+    emailError.classList.toggle('hide');
+  }
+});
```

Comment:
- [ ] Great job adding validations for all your fields 💯 but this project required only the email validation. If the email is invalid an error message should appear near the submit button and if the email is valid the form should be sent. Right now your email validation is appearing under the email input. Kindly make the email validation appear close to the submit button as per the requirements and you'll be good to go 👍 
- [ ] Also, as the name and message validations are not needed, so it would be good to remove those so as to follow the requirements 😄 
- Requirements:
![Annotation 2022-04-08 051110](https://user-images.githubusercontent.com/85113567/162361209-94aa34f7-6e77-4052-a79e-32bdf91af574.jpg)
- Your design:
![Annotation 2022-04-08 050956](https://user-images.githubusercontent.com/85113567/162361228-ffd9cd24-debf-42cd-9bb5-ef0b7ae55171.jpg)

Reply:
- Make email validation appear close to the submit button as per the requirements 👍🏽 
- Also, as the name and message validations are not needed, it would be good to remove those so as to follow the requirements 👍🏽. #Removed. ✔️